### PR TITLE
feat(routes): Initiate app, Add routes, debug models, config settings…

### PR DESCRIPTION
**Pull request by senthil2000-dev**
…, make migrations, add template views

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,3 +18,7 @@ pgAdmin interface is served at localhost:5050 with username admin@admin.com and
 2. `source ./venv/bin/activate`
 3. `pip install -r requirements.txt`
 4. `pip install pre-commit pylint && pre-commit install`
+
+## To load departments and privileges from fixtures
+`python manage.py loaddata api/fixtures/Department.json --app api.Department`
+`python manage.py loaddata api/fixtures/ProjectMemberPrivilege.json --app api.ProjectMemberPrivilege`
```
Comment (subramaniyam73) [inline README.md:24]: Change the command to load everything from fixtures

File: api/controllers/project_utilities.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from api.models import Project, ProjectMemberPrivilege, ProjectMemberRelationship
+import logging
+
+logger = logging.getLogger(__name__)
+
+def create_project(name, abstract, paper_link, head, department, aor):
```
Comment (subramaniyam73) [inline api/controllers/project_utilities.py:6]: Add docstring

Comment (senthil2000-dev) [reply api/controllers/project_utilities.py:6]: done

File: README.md (hunk 1 above)
Comment (senthil2000-dev) [reply README.md]: done

**Review by marmikupadhyay — APPROVED** [review]
lgtm, changes told by subbu done.

**Review by marmikupadhyay — APPROVED** [review]

**Review by subramaniyam73 — APPROVED** [review]
Changes completed

**Review by kelpikz — APPROVED** [review]

**Pull request merged**